pkg/kubelet/images/image_manager.go

```diff
@@ -101,7 +101,18 @@ func (m *imageManager) EnsureImageExists(pod *v1.Pod, container *v1.Container, p
 		return "", msg, ErrInvalidImageName
 	}
 
-	spec := kubecontainer.ImageSpec{Image: image}
+	var podAnnotations []kubecontainer.Annotation
+	for k, v := range pod.GetAnnotations() {
+		podAnnotations = append(podAnnotations, kubecontainer.Annotation{
+			Name:  k,
+			Value: v,
+		})
+	}
+
+	spec := kubecontainer.ImageSpec{
+		Image:       image,
+		Annotations: podAnnotations,
```

Comment:
Is there annotation that needs to be filtered before passing to ImageSpec ?

Reply:
It won't hurt anything to include all Pod annotations here and let the CRI/containerd pick which ones to use so I figured it would be cleaner here to just pass them all along.
I can update to include filtering if necessary - but that would require code updates in k/k to pass along other annotations in the future.
What does everyone think?

Reply:
I think it's hard to determine which annotations should be filtered. let's just pass all?